plugins/woocommerce-admin/client/products/fields/attribute-term-input-field/test/attribute-term-input-field.spec.tsx

```diff
@@ -75,80 +75,81 @@ jest.mock( '@woocommerce/components', () => {
 	};
 } );
 
-const attributeList: ProductAttribute[] = [
-	{
-		id: 15,
-		name: 'Automotive',
-		position: 0,
-		visible: true,
-		variation: false,
-		options: [ 'test' ],
-	},
-	{
-		id: 1,
-		name: 'Color',
-		position: 2,
-		visible: true,
-		variation: true,
-		options: [
-			'Beige',
-			'black',
-			'Blue',
-			'brown',
-			'Gray',
-			'Green',
-			'mint',
-			'orange',
-			'pink',
-			'Red',
-			'white',
-			'Yellow',
-		],
-	},
-];
+// Not used - TODO: Remove?
```

Comment:
I tried to avoid doing too much of this kind of thing but occasionally it felt right to do.